### PR TITLE
Move Windows deployment to stable overlay

### PR DESCRIPTION
**Pull request by jingxu97**
move Windows pd csi deployent to stable overlay

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: deploy/kubernetes/base/node_windows/node.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
         kubernetes.io/os: windows
       containers:
         - name: csi-driver-registrar
-          image: gke.gcr.io/k8s-staging-csi/csi-node-driver-registrar-win
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar
```
Comment (mattcary) [inline deploy/kubernetes/base/node_windows/node.yaml:27]: Can you explain why we don't have to use a windows image for the registrar? Do we have a multi-arch build or something like that?

(and ditto for the driver image)

Comment (jingxu97) [reply deploy/kubernetes/base/node_windows/node.yaml:27]: right, both container images support multi-arch build now.

**Comment by mattcary** [comment]
Sorry, this got lost in my inbox.

/lgtm

**Pull request merged**